secure_message/validation/user.py

```diff
@@ -17,6 +17,11 @@ def is_respondent(self):
         return bool(self.role == 'respondent')
 
     @staticmethod
-    def is_valid_user(uuid):
+    def is_valid_internal_user(uuid):
+        _, status_code = internal_user_service.get_user_details(uuid)
+        return True if status_code == 200 else False
```

Comment:
can be simplified to

return status_code == 200

Reply:
doh , thanks , done